modules/exploits/windows/local/ms10_092_schelevator.rb

```diff
@@ -298,39 +293,40 @@ def fix_crc32(data, old_crc)
     crc = crc32(data[0, data.length - 12])
     data[-12, 4] = [crc].pack('V')
 
-    data[-12, 12].unpack('C*').reverse.each { |b|
+    data[-12, 12].unpack('C*').reverse.each do |b|
       old_crc = ((old_crc << 8) ^ bwd_table[old_crc >> 24] ^ b) & 0xffffffff
-    }
+    end
     data[-12, 4] = [old_crc].pack('V')
   end
 
-  def exec_schtasks(cmdline, purpose)
-    cmdline = "/c #{cmdline.strip}"
-    lns = cmd_exec('cmd.exe', cmdline)
-
-    success = false
-    lns.each_line { |ln|
+  def exec_schtasks(cmdline)
```

Comment:
It looks like with the exception of the `change` verb, the rest of the uses of this function could be replaced with the [new mixin](https://github.com/rapid7/metasploit-framework/blob/master/lib/msf/core/post/windows/task_scheduler.rb#L9).

Reply:
done